src/servo/components/style/gecko/media_features-rs.patch

```diff
@@ -0,0 +1,54 @@
+diff --git a/servo/components/style/gecko/media_features.rs b/servo/components/style/gecko/media_features.rs
+index 2baf0df323c83107a9a38e76bda3a273892b121e..f14fc90e048534f7345be102783e0669fa03ae75 100644
+--- a/servo/components/style/gecko/media_features.rs
++++ b/servo/components/style/gecko/media_features.rs
+@@ -9,10 +9,13 @@ use crate::gecko_bindings::structs;
+ use crate::media_queries::{Device, MediaType};
+ use crate::queries::feature::{AllowsRanges, Evaluator, FeatureFlags, QueryFeatureDescription};
+ use crate::queries::values::Orientation;
++use crate::queries::condition::MozPrefFeatureValue;
```

Comment:
Could all of this be joined into a line-by-line (next to each other)? To avoid having the patch extend over multiple lines